DOCS/man/options.rst

```diff
@@ -2440,6 +2443,14 @@ Subtitles
     times to add multiple files. Technically, ``--sub-file`` is a CLI/config
     file only alias for  ``--sub-files-append``.
 
+``--sub-file-priority=<yes|no|never>``
+    When loading any type of external subtitle track, choose whether to
+    prioritize its autoselection over other potential aspects of the track
+    (e.g. slang or various --subs-fallback options). This defaults to ``yes``
+    meaning that external subtitle files have priority over internal subtitles.
+    ``no`` removes any special prioritization whereas ``never`` will prefer
+    internal tracks.
```

Comment:
> whereas ``never`` will prefer internal tracks.

never should more like "never select". This doesn't sound like it.

Reply:
I'm not sure. It's more like `deprioritize` but that sounds weird.

Reply:
How about just call it `inter` ...

Reply:
Did you typo? Not sure what you meant. I was thinking maybe `reverse` or `invert` but that also sounds weird.